src/protocol/tx-validity.md

```diff
@@ -212,14 +212,14 @@ def max_gas(tx) -> int:
     if tx.type == TransactionType.Script:
        gas = gas + tx.gasLimit
     return gas
-
-
-def reserved_feeBalance(tx, assetId) -> int:
```

Comment:
I couldn't find any function with this name in our code, and it sounds like `max_fee`, so I changed it. Might be wrong.

Reply:
Yeah, in the code analog is `max_fee` function=) Could you also update other places here(`reserved_fee_balance`), please?